### ISSUE TITLE
Add support for Fancy Holograms

### ISSUE THREAD

**Issue by Jazzkuh** [SkytAsul/BeautyQuests#322]
Would it be possible to add support for FancyHolograms? https://github.com/FancyMcPlugins/FancyHolograms/tree/f2fb8d675c1dd056d2936d7226b11ba54858b158

**Comment by SkytAsul** [comment]
From what I see, it's not possible to have a hologram with both text and items. It may be a limitation.

**Comment by Jazzkuh** [comment]
> From what I see, it's not possible to have a hologram with both text and items. It may be a limitation.

Yes true, but it think for servers wanting to use the new text displays instead of traditional holograms and not using 2 hologram plugins it would be better to have support for it on BeautyQuests.

**Comment by Jazzkuh** [comment]
> Sure. If I have time I will add this integration. Though, if you know a developer that could do it, the API is free to use and pull requests are welcome :)

Cool thanks!
